zhaquirks/tuya/air/ts0601_air_quality.py

```diff
@@ -47,7 +46,7 @@ class TuyaCO2Sensor(CustomDevice):
                 ],
                 OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
             }
-        },
+        }
```

Comment:
Don't change.
Black complains about it:
* https://github.com/zigpy/zha-device-handlers/runs/5167595067?check_suite_focus=true